examples/server/server.cpp

```diff
@@ -28,9 +28,9 @@ class ServerImpl : public Server {
   std::vector<std::thread> threads;
 
 public:
-  INJECT(ServerImpl()) {}
+  INJECT(ServerImpl()) = default;
 
-  ~ServerImpl() {
+  ~ServerImpl() override {
```

Comment:
The `override` here isn't really necessary, Fruit deallocates objects by calling the destructor of the concrete class (as opposed to relying on the virtual destructor in the base class).
However it's possible that some compilers might complain about the lack of virtual/override.
What was the motivation for adding this?

Reply:
Because `Server` destructor is `virtual`, so this is the proper way to mark an overriden destructor. Furthermore, removing the keyword won't make it non-overriden

Reply:
Sorry. I marked `Server` as `virtual`.